### ISSUE TITLE
If Actor Facing event doesn't recognize UP or LEFT

### ISSUE THREAD

**Issue by timsamoff** [chrismaltby/gb-studio#218]
**Describe the bug**
If Actor Facing event doesn't recognize UP or LEFT...

**To Reproduce**
1. Add an "If Actor Facing" event to a scene and set it to Up or LEFT
2. Within that event, set a variable to a value (such as 1)
3. Setup a Display Dialogue event to output the value of that variable.
4. The result will be "0" (null).
5. If you do this same thing with RIGHT or DOWN, the variable's value will be output correctly.

**Expected behavior**
For all player directions to be recognized by the If Actor Facing Event.

**Screenshots**
<img width="480" alt="Screen Shot 2019-06-15 at 08 37 38" src="https://user-images.githubusercontent.com/62411/59553375-e261fa80-8f48-11e9-8cf1-eada2dde9fa2.png">

**Platform (please complete the following information):**
 - OS: macOS 10.14.5
 - App version 1.1.0

**Comment by chrismaltby** [comment]
Hi Tim, I'm unable to reproduce this (also on macOS). Could you maybe upload a small sample project with the problem to investigate further? Thanks

**Comment by timsamoff** [comment]
I'm assuming that this error is mine, but here's what I'm trying right now:

[dir_bug.zip](https://github.com/chrismaltby/gb-studio/files/3346993/dir_bug.zip)

Btw, I've also tried this by nesting the "ifs" in "elses."

The idea is that when you go a certain direction, a variable will be stored. If you hit the B button, a dialogue box will appear that displays the stored variable.

Currently, it seems as if only the player's starting direction is stored.

My directions/variables are set up like this:
North: 1
East: 2
South: 3
West: 4

Please let me know what I'm doing wrong.

**Comment by chrismaltby** [comment]
Hi Tim,

The way your project was set up you were checking the player's direction but only as the scene initially loaded and then you were displaying the dialogue based on that initial value.

So whatever direction the player was facing as the scene loaded was the value that would always be returned. All you needed to do was move your "Ifs" into the "Attach Script To Button" part so that the direction is checked every time you press the button, that seems to work for me then.

I've attached a fixed project (looks like you've updated to the 1.2.0+ dev builds so I've kept it in that format).
[dir_bug_fixed.zip](https://github.com/chrismaltby/gb-studio/files/3355077/dir_bug_fixed.zip)

Hope this helps.
